.pipelines/ci.yml

```diff
@@ -1,20 +1,24 @@
 # Azure DevOps Pipeline running CI
 trigger:
-- master
+  branches:
+    include:
+    - master
+  paths:
+    exclude:
+    - docs/*
```

Comment:
Changes the implicit default from `master` branch to explicit `master` branch with omit of `docs/*` folder. 

Documentation for triggers: https://docs.microsoft.com/en-us/azure/devops/pipelines/yaml-schema?view=azure-devops&tabs=schema%2Cparameter-schema#push-trigger